.github/disabled/e2e-gatsbyjs-cli-workflow.yml

```diff
@@ -9,7 +9,7 @@ name: 'E2E Gatsby.js CLI with verdaccio'
 jobs:
   npm:
     name: 'npm:gatsby example'
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
```

Comment:
Isn't the matrix setup still missing here?

Reply:
Nvm, seems the branch is updated now.

Reply:
yes, I think so
I'll update the configs